content/en/docs/ops/common-problems/validation/index.md

```diff
@@ -147,24 +147,24 @@ configuration. Verify that it is not empty (see [verify webhook
 configuration](#invalid-configuration-is-accepted)). Istio consciously reconciles webhook configuration
 used the `istio-validation` `configmap` and root certificate.
 
-1. Verify the `istio-pilot` pod(s) are running:
+1. Verify the `istiod` pod(s) are running:
 
     {{< text bash >}}
-    $  kubectl -n istio-system get pod -lapp=pilot
+    $  kubectl -n istio-system get pod -lapp=istiod
     NAME                            READY     STATUS    RESTARTS   AGE
-    istio-pilot-5dbbbdb746-d676g   1/1       Running   0          2d
+    istiod-5dbbbdb746-d676g   1/1       Running   0          2d
     {{< /text >}}
 
 1. Check the pod logs for errors. Failing to patch the
        `caBundle` should print an error.
 
     {{< text bash >}}
-    $ for pod in $(kubectl -n istio-system get pod -lapp=pilot -o jsonpath='{.items[*].metadata.name}'); do \
+    $ for pod in $(kubectl -n istio-system get pod -lapp=istiod -o jsonpath='{.items[*].metadata.name}'); do \
         kubectl -n istio-system logs ${pod} \
     done
     {{< /text >}}
 
-1. If the patching failed, verify the RBAC configuration for Pilot:
+1. If the patching failed, verify the RBAC configuration for `Istiod`:
```

Comment:
From looking at the release announcements PR (https://github.com/istio/istio.io/pull/6705), it seems we are using `istiod`. I'm wondering if this should be updated to match. I know I asked this previously, just didn't see any official response on what it should be at this point.

Reply:
Alright, per some discussion we should be uppercasing Istiod when talking about the component and leaving lower case when talking about the deployment. So, can we just remove that last commit where I asked for the backticks and then we'll wait for the PR updating the dictionary with Istiod and then this should get through the checks.

Thanks John.